consensus/state_test.go

```diff
@@ -523,7 +523,10 @@ func TestLockPOLRelock(t *testing.T) {
 	<-voteCh // prevote
 
 	signAddVotes(cs1, types.VoteTypePrevote, cs1.ProposalBlock.Hash(), cs1.ProposalBlockParts.Header(), vs2, vs3, vs4)
-	_, _, _ = <-voteCh, <-voteCh, <-voteCh // prevotes
+	// prevotes
+	<-voteCh
```

Comment:
a bit hard to read... easier than the other style though.

Why not `discardN(chan, int)` that grabs N messages from a channel and discard them.  Seems to be used all over this file?